modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/extension/v1_0/test/ObjectEntryExtensionProviderTest.java

```diff
@@ -0,0 +1,231 @@
+import com.liferay.portal.vulcan.extension.ExtensionProvider;
+import com.liferay.portal.vulcan.extension.PropertyDefinition;
+
+import java.io.Serializable;
+
+import java.math.BigDecimal;
+
+import java.sql.Timestamp;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.Map;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Carolina Barbosa
+ */
+@RunWith(Arquillian.class)
+public class ObjectEntryExtensionProviderTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
+		new LiferayIntegrationTestRule();
+
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		PropsUtil.addProperties(
+			UnicodePropertiesBuilder.setProperty(
+				"feature.flag.LPS-135404", "true"
+			).build());
+
+		ObjectDefinition objectDefinition =
+			_objectDefinitionLocalService.fetchObjectDefinitionByClassName(
+				TestPropsValues.getCompanyId(), User.class.getName());
+
+		_addCustomObjectField(
+			objectDefinition.getObjectDefinitionId(), "Boolean", "Boolean",
+			"booleanField", false);
+		_addCustomObjectField(
+			objectDefinition.getObjectDefinitionId(), "Date", "Date",
+			"dateField", true);
+		_addCustomObjectField(
+			objectDefinition.getObjectDefinitionId(), "Decimal", "Double",
+			"decimalField", false);
+		_addCustomObjectField(
+			objectDefinition.getObjectDefinitionId(), "PrecisionDecimal",
+			"BigDecimal", "precisionDecimalField", true);
+
+		_user = UserTestUtil.addUser();
+	}
+
+	@AfterClass
+	public static void tearDownClass() throws Exception {
+		PropsUtil.addProperties(
+			UnicodePropertiesBuilder.setProperty(
+				"feature.flag.LPS-135404", "false"
+			).build());
+	}
+
+	@Test
+	public void testGetExtendedPropertyDefinitions() throws Exception {
+		Map<String, PropertyDefinition> extendedPropertyDefinitions =
+			_extensionProvider.getExtendedPropertyDefinitions(
+				TestPropsValues.getCompanyId(), UserAccount.class.getName());
+
+		_assertPropertyDefinition(
+			extendedPropertyDefinitions.get("booleanField"), "booleanField",
+			PropertyDefinition.PropertyType.BOOLEAN, false);
+		_assertPropertyDefinition(
+			extendedPropertyDefinitions.get("dateField"), "dateField",
+			PropertyDefinition.PropertyType.DATE_TIME, true);
+		_assertPropertyDefinition(
+			extendedPropertyDefinitions.get("decimalField"), "decimalField",
+			PropertyDefinition.PropertyType.DOUBLE, false);
+		_assertPropertyDefinition(
+			extendedPropertyDefinitions.get("precisionDecimalField"),
+			"precisionDecimalField",
+			PropertyDefinition.PropertyType.BIG_DECIMAL, true);
+	}
+
+	@Test
+	public void testIsApplicableExtension() throws Exception {
+		Assert.assertFalse(
+			_extensionProvider.isApplicableExtension(
+				TestPropsValues.getCompanyId(), null));
+		Assert.assertTrue(
```

Comment:
Can we include another assert to make sure a custom definition is false?

Reply:
Done!

Reply:
Thanks 🙏🏻